### ISSUE TITLE
qtile shell : error generated by 'cd' command.

### ISSUE THREAD

**Issue by Fabien-Bernard** [qtile/qtile#2294]
# Issue description
My qtile shell ended with an error after I type a "cd" command without a path.
I was expected to go back to the root or to get a "no such path" message.

# Qtile version
git show-ref v0.17.0
fcde02230915b603c36f391deb78db2c0ca35a08 refs/tags/v0.17.0

# Stack traces
```
[bernard@vmArchqTile ~]$ qtile shell
 > ls
bar/     group/   layout/  screen/  widget/  window/
 > cd bar
bar
bar > ls
top/
bar > cd top/
bar[top]
bar[top] > help
help command   -- Help for a specific command.

Builtins
========
cd    exit  help  ls    pwd   q     quit

Commands for this object
========================
commands           doc                eval               fake_button_press
function           info               items            
bar[top] > cd titi
No such path.
bar[top] > cd
Traceback (most recent call last):
  File "/usr/bin/qtile", line 33, in <module>
    sys.exit(load_entry_point('qtile==0.17.0', 'console_scripts', 'qtile')())
  File "/usr/lib/python3.9/site-packages/libqtile/scripts/main.py", line 59, in main
    options.func(options)
  File "/usr/lib/python3.9/site-packages/libqtile/scripts/shell.py", line 35, in qshell
    qsh.loop()
  File "/usr/lib/python3.9/site-packages/libqtile/sh.py", line 348, in loop
    val = self.process_line(line)
  File "/usr/lib/python3.9/site-packages/libqtile/sh.py", line 308, in process_line
    val = builtin(args)
  File "/usr/lib/python3.9/site-packages/libqtile/sh.py", line 209, in do_cd
    next_node = self._find_path(arg)
  File "/usr/lib/python3.9/site-packages/libqtile/sh.py", line 152, in _find_path
    root = graph.CommandGraphRoot() if path.startswith("/") else self._current_node
AttributeError: 'NoneType' object has no attribute 'startswith'
[bernard@vmArchqTile ~]$
```

# Configuration
Not relevant I think.

**Comment by m-col** [comment]
Interestingly I'm not getting the full traceback, the error appears to be caught by the main script, which continues to print out the usage message:
```
 > cd
usage: qtile [-h] [-v] {start,shell,top,run-cmd,cmd-obj,check,migrate,help} ...

Did you mean:
./bin/qtile shell start
```
You getting this on current master (b4dd1cbbe47f91510fdad3ad2c7159a668f01e6f)?

**Comment by Fabien-Bernard** [comment]
No, I don't think so. 
I first installed qtile a few weeks ago with the command "pacman -S qtile" (arch distribution). 
The command "qtile --version" output is 0.17.0. 
In the description above, I described how I tried to retrieve the right commit with this tag. Sorry, I lack praticed on how to retrieve this infomation or how to easily change the version of my qtile depending of a particular commit.

After your remark, I installed a new version of qtile with the command "pacman -S qtile-git" this time and I have now the same output than you.
So if I understand well, I corrected a no more existing bug. 
Thank you for pointing that out, at least I learned that there is something wrong the way I do things.
